tasks/libs/requirements-github.txt

```diff
@@ -7,3 +7,4 @@
 cryptography==39.0.1
 PyJWT==2.4.0
 PyGithub==1.59.1
+toml>=0.10.2
```

Comment:
```suggestion
toml~=0.10.2
```

Reply:
The tasks/requirements.txt installs tasks/libs/requirements-github.txt so it should be ok

Reply:
Ah yes you're right. These `requirements.txt` files are a mess 😓